front/prompt/global_agent_helper_prompt.md

```diff
@@ -6,115 +6,174 @@ Make sure you answer with all the details. Double-check your answers for errors,
 
 # About Dust
 
-### What are Dust assistants?
+## What is Dust?
 
-Assistants are powered by GPT4 and Claude. It's designed to help teams work better with AI. These AI assistants are there to enhance your understanding of any topic, increase productivity, and improve work processes. They can help with company procedures, draft memos, or simplify complex topics.
+Conversation is a Dust product powered by GPT4 and Claude. It's designed to help teams work better with AI. These AI agents are there to enhance your understanding of any topic, increase productivity, and improve work processes. They can help with company procedures, draft memos, or simplify complex topics.
 
-### What can I use an assistant for?
+## General concepts
 
-A Dust assistant can answer questions and chat with you. Each one is different, so check their descriptions on the conversation homepage to see which one(s) to use.
+### Assistant
 
-Use @dust for general company questions. It has access to your company data as well as public data up to September 2021 thanks to GPT4
+Dust assistants are AI-powered agents that employ advanced models like GPT-4 and Claude. Their purpose is to assist teams by expanding comprehension, boosting productivity and creativity, and refining workflows. Each assistant possesses unique capabilities to answer questions, assist with company processes, draft emails and memos, unravel complex topics, and perform many other customized functions created by builders. For example while @Dust handles organizational questions, @helper provides Dust support, @slack searches Slack, and @gpt4/@claude offer direct large language model access. Multiple Assistants can be leveraged concurrently to tackle varied tasks.
```

Comment:
what does "refining workflows" mean?

Reply:
@Dust -> @dust

Reply:
refining workflows = Optimize and simplify the processes by which a task or job is accomplished. no? 

Tks for catching this dust

Reply:
github displays @xxx without uppercase but I meant @ Dust > @ dust to be clear (you'll figure out the dozen other :))

Reply:
> refining workflows = Optimize and simplify the processes by which a task or job is accomplished. no?

Well i'd say just that. Refining workflow does not make a ton of sense to me FYI